projects/macvim.org/package.yml

```diff
@@ -0,0 +1,82 @@
+distributable:
+  url: https://github.com/macvim-dev/macvim/archive/release-{{version.major}}.tar.gz
+  strip-components: 1
+versions:
+  github: macvim-dev/macvim/tags
+  match: /release-\d+/
+  strip:
+    - /^release-/
+platforms:
+  - darwin
+dependencies:
+  cscope.sourceforge.io: '*'
+  lua.org: '*'
+  python.org: ^3.11
+  ruby-lang.org: '*'
+  invisible-island.net/ncurses: '*'
+runtime:
+  env:
+    PYTHONHOME: "{{deps.python.org.prefix}}"
+    DYLD_LIBRARY_PATH: "{{deps.python.org.prefix}}/lib/libncurses.6.dylib:$DYLD_LIBRARY_PATH"
```

Comment:
This doesn't look right; wouldn't it be `deps.invisible-island.net/ncurses/prefix`? I also worry about setting `PYTHONHOME`.